movie/src/components/MovieList/index.jsx

```diff
@@ -2,18 +2,74 @@ import "./MovieList.css";
 import Navbar from "../Navbar";
 import MainContainer from "../MainContainer";
 import movies from "../../data/Movies.json";
+import Footer from "../Footer";
+import DisplayPageViaRoute from "../DisplayPageViaRoute";
+
+import NowPlaying from "../../pages/NowPlaying";
+import TopRated from "../../pages/TopRated";
+import UpComing from "../../pages/UpComing";
+import PopularTv from "../../pages/PopularTv";
+import AiringToday from "../../pages/AiringToday";
+import OnTv from "../../pages/OnTv";
+import TopRate from "../../pages/TopRate";
+import PopularPeople from "../../pages/PopularPeople";
+import Discussion from "../../pages/Discussion";
+import Leaderboard from "../../pages/LeaderBoard";
+import Support from "../../pages/Support";
+import ApiMore from "../../pages/ApiMore";
+
+import AboutTmdb from "../../pages/AboutTmdb";
+import ContactUs from "../../pages/ContactUs";
+import SupportForums from "../../pages/SupportForums";
+import ApiBasic from "../../pages/ApiBasic";
+import SystemStatus from "../../pages/SystemStatus";
+import ContributionBible from "../../pages/ContributionBible";
+import AddNewMovie from "../../pages/AddNewMovie";
+import AddNewTvShow from "../../pages/AddNewTvShow";
+import Guidlines from "../../pages/GuideLines";
+import Discussions from "../../pages/Discussions";
+import LeaderBoardCommunity from "../../pages/LeaderBoardCommunity";
+import Twitter from "../../pages/Twitter";
+import TermsOfUse from "../../pages/TermsOfUse";
+import ApiTermsOfUse from "../../pages/ApiTermsOfUse";
+import PrivacyPolicy from "../../pages/PrivacyPolicy";
 
 const MovieList = () => {
+  const pagesRouters = [
```

Comment:
why are we doing `routing` in a `MovieList` file?

Reply:
Because movie list is the main file which collect of navbar, containers and footer. it is my fault which I name the file not suitable.